run_meta_analysis.py

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env python
+# coding: utf-8
+import argparse
+import hail as hl
+from ukbb_pan_ancestry import *
+from ukb_common import *
+
+
+def all_and_leave_one_out(x, pop, all_f=hl.sum, loo_f=lambda i, x: hl.sum(x) - hl.or_else(x[i], 0)):
```

Comment:
This looks great! Might want to document the parameters a bit in a docstring and/or spell them out (e.g., `pop` -> `pop_array`).

```
def all_and_leave_one_out(...):
    """
    What the function does

    :param x: Description of x
    :param pop: Description of pop
    ...
    :return: Description of return
    :rtype: ArrayExpression
    """
    arr = ...
```

Reply:
Added